src/js/UrbanAirship.js

```diff
@@ -175,14 +175,13 @@ class UrbanAirship {
 
   /**
    * Global data collection flag. Enabled by default, unless `dataCollectionOptInEnabled`
-   * is set to `YES` in AirshipConfig.
+   * is set to `YES` in AirshipConfig.plist on iOS, and `true` in airshipconfig.properties on Android.
    * When disabled, the device will stop collecting and sending data for named user, events,
    * tags, attributes, associated identifiers, and location from the device.
    *
-   * Push notifications will continue to work only if `UrbanAirshipModule.setPushTokenRegistrationEnabled`
-   * has been explicitly set to `YES`, otherwise it will default to the current state  of `isDataCollectionEnabled`.
+   * Push notifications will continue to work only if `UrbanAirship.setPushTokenRegistrationEnabled`
```

Comment:
I think you lost a line here:
```suggestion
   * Push notifications will continue to work only if `UrbanAirship.setPushTokenRegistrationEnabled`
   * has been explicitly set to `YES`, otherwise it will default to the current state  of `isDataCollectionEnabled`.
```